README.md

```diff
@@ -19,13 +19,13 @@
 | [cert-manager-cainjector](./images/cert-manager-cainjector) | `cgr.dev/chainguard/cert-manager-cainjector` | experimental | [![](https://storage.googleapis.com/chainguard-images-build-outputs/badges/cert-manager-cainjector.build.status.latest-dev.svg)](https://registry-ui.chainguard.app/?image=cgr.dev/chainguard/cert-manager-cainjector:latest-dev)<br/>[![](https://storage.googleapis.com/chainguard-images-build-outputs/badges/cert-manager-cainjector.build.status.latest.svg)](https://registry-ui.chainguard.app/?image=cgr.dev/chainguard/cert-manager-cainjector:latest) |
 | [cert-manager-controller](./images/cert-manager-controller) | `cgr.dev/chainguard/cert-manager-controller` | experimental | [![](https://storage.googleapis.com/chainguard-images-build-outputs/badges/cert-manager-controller.build.status.latest-dev.svg)](https://registry-ui.chainguard.app/?image=cgr.dev/chainguard/cert-manager-controller:latest-dev)<br/>[![](https://storage.googleapis.com/chainguard-images-build-outputs/badges/cert-manager-controller.build.status.latest.svg)](https://registry-ui.chainguard.app/?image=cgr.dev/chainguard/cert-manager-controller:latest) |
 | [cert-manager-webhook](./images/cert-manager-webhook) | `cgr.dev/chainguard/cert-manager-webhook` | stable | [![](https://storage.googleapis.com/chainguard-images-build-outputs/badges/cert-manager-webhook.build.status.latest-dev.svg)](https://registry-ui.chainguard.app/?image=cgr.dev/chainguard/cert-manager-webhook:latest-dev)<br/>[![](https://storage.googleapis.com/chainguard-images-build-outputs/badges/cert-manager-webhook.build.status.latest.svg)](https://registry-ui.chainguard.app/?image=cgr.dev/chainguard/cert-manager-webhook:latest) |
-| [clang](./images/clang) | `cgr.dev/chainguard/clang` | experimental | [![](https://storage.googleapis.com/chainguard-images-build-outputs/badges/clang.build.status.latest-dev.svg)](https://registry-ui.chainguard.app/?image=cgr.dev/chainguard/clang:latest-dev)<br/>[![](https://storage.googleapis.com/chainguard-images-build-outputs/badges/clang.build.status.latest.svg)](https://registry-ui.chainguard.app/?image=cgr.dev/chainguard/clang:latest) |
+| [clang](./images/clang) | `cgr.dev/chainguard/clang` | experimental | [![](https://storage.googleapis.com/chainguard-images-build-outputs/badges/clang.build.status.latest.svg)](https://registry-ui.chainguard.app/?image=cgr.dev/chainguard/clang:latest) |
```

Comment:
is there anyway to resolve missing dev variants?

Reply:
Unclear, but it's one of several things we will need an answer to before we land this.  Others on my list:
1. Rumble (could potentially key off of `chainctl img ls`?)
2. Image summaries (same?)

In general, I wonder how much we should lean on markdown for discovery of images going forwards vs. leaning into the catalog UI and driving requirements there.  cc @pdeslaur @amdawson